python_modules/dagster/dagster/_core/storage/schedules/schema.py

```diff
@@ -45,6 +47,24 @@
     db.Column("update_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetPolicyEvaluationsTable = db.Table(
+    "asset_policy_evaulations",
+    ScheduleStorageSqlMetadata,
+    db.Column(
```

Comment:
what's the difference between id and evaluation_id?

Reply:
id is a primary key assigned by the db
evaluation id is the number for the daemon iteration. It's non-unique